### PR TITLE
3rd Draft of the "Shards Flow" Chapter

### PR DESCRIPTION
**Pull request by ChelseaChX**
- Added figures
- Fixed some formatting
- Added the segment on Stepmany, Branch and TryMany

### CODE REVIEW

**Comment by sinkingsugar** [comment]
how to review it properly? Is there a temporary page?

**Comment by ChelseaChX** [comment]
> how to review it properly? Is there a temporary page?

Currently we're using mkdocs to preview with a static site. I can put it on GitBook so that it is easier for you and Guus to review. I'll link the page here once I've moved the content over.

**Comment by sinkingsugar** [comment]
> > how to review it properly? Is there a temporary page?
> 
> Currently we're using mkdocs to preview with a static site. I can put it on GitBook so that it is easier for you and Guus to review. I'll link the page here once I've moved the content over.

I definitely cannot go to the extent do setup mkdocs, so something ready to consume is needed

**Comment by ChelseaChX** [comment]
@sinkingsugar @guusw 

Here is the gitbook link for review. Thank you!

https://app.gitbook.com/o/3P6QfbtIO6GdyTCAishv/s/kU8RKpFJAomTkF4Ipcxy/the-flow

File: docs/docs/learn/shards/the-flow.md [hunk 1]
```diff
@@ -55,14 +77,27 @@ In the example below, John starts with five apples. He wishes to share them equa
 
 [`Detach`](../../../reference/shards/General/Detach) and [`Spawn`](../../../reference/shards/General/Spawn) schedules a Wire to run on the same Mesh.
 
-The difference between `Detach` and `Spawn` is that `Detach` schedules the original Wire itself, while `Spawn` schedules clones of the Wire. This means that there can only be one instance of the detached Wire running, while you can have many instances of the spawned Wire.
+The difference between `Detach` and `Spawn` is that:
+
+- `Detach` schedules the original Wire itself.
+
+- `Spawn` schedules clones of the Wire.
```
Comment (guusw) [inline docs/docs/learn/shards/the-flow.md:84]: "Spawn schedules a clone of the wire" sounds better I think

File: docs/docs/learn/shards/the-flow.md [hunk 2]
```diff
@@ -422,20 +465,197 @@ The example also showcases how variables defined in `john` are affected by chang
...
+    [take-apple-5] Apples left: 86
+    ```
+
+## Branch
+
+[`Branch`](../../../reference/shards/General/Branch) is used when you wish to create a Submesh on the current Mesh. You can schedule Wires on the Submesh by placing Wires in its `Wires` parameter. These Wires will behave as if they were run with `Step`.
+
+=== "Syntax"
+    ```{.clojure .annotate linenums="1"}
+    (Branch [wire-x, wire-y, wire-z]) ;; (1)
```
Comment (guusw) [inline docs/docs/learn/shards/the-flow.md:642]: Remove `,`

**Review by guusw — CHANGES_REQUESTED** [review]
Still reading over Spawn/Resume

**Comment by ChelseaChX** [comment]
@guusw I've removed the mentions on Dispatch and integrated the comments given. I've left out mentions of Passthrough for now, will be adding that to the following chapter.

**Pull request merged**